recipes/blockbuster/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+make
+mkdir $PREFIX/bin
+cp $SRC_DIR/blockbuster.x $PREFIX/bin
```

Comment:
I just checked it. It is like Kyle mentions. While the executable is still visible in the test environment, it is not anymore after installing the package.